### PR TITLE
🐞⚡Fix opengraph and twitter tags method + add mercury snap

### PR DESCRIPTION
**Pull request by FaazAbidi**
#### GET endpoint
- Fixed opengraph tags method. Some edge cases weren't passing.
- Fixed twitter tags method. Added another key for searching inside twitter tags which is 'name'. 
- Add a new API for hosting screenshots images called imgbb
- Created and added our own screenshots API called mercury snap. Here's the repo for that: [Mercury Snap](https://github.com/FaazAbidi/mercury-snap). Created this over the weekend mainly for these reasons:
        - It's faster than other pip packages
        - It's free! other APIs have pricing issues. API flash was limited to 100 screenshots/month. 
        - Scalable. We will add more features a/c to our needs.

### CODE REVIEW

File: utils/enrichment.py [hunk 1]
```diff
@@ -38,7 +37,7 @@ def enrich_test(url: str) -> Dict:
         elif site_name == "www.artsy.net" or site_name == "artsy.net":
             enrich_data = fetch_artsy(url)
 
-        elif site_name == "www.wikiart.org":
+        elif site_name == "www.wikiart.org" or site_name == "wikiart.org":
```
Comment (hxghhhh) [inline utils/enrichment.py]:
```suggestion
        elif site_name in ('www.wikiart.org', 'wikiart.org'):
```

File: utils/tag_parsers.py [hunk 2]
```diff
@@ -38,36 +38,47 @@ def medium_check(get_data, form_type) -> str:
 
 
 # getting Open graph tags data
-def open_graph(request_object, get_data) -> dict:
+def open_graph(tupl) -> dict:
+    request_object = tupl[0]
+
+    get_data = {
```
Comment (hxghhhh) [inline utils/tag_parsers.py:48]: use `defaultdict` instead adding all these entries upfront

Comment (FaazAbidi) [reply utils/tag_parsers.py:48]: Okay, I'll do that 👍

**Review by joelrojo — APPROVED** [review]
@FaazAbidi great work on this, love that you took the initiative to create your own service for taking screenshots, that's so clutch!! 

giphy (2)

**Pull request merged**